diagrams/get-access/civil-legal-aid-system-landscape.yaml

```
@@ -43,7 +43,7 @@ elements:
   tags: external
   position: '700,400'
 - type: Software System
-  name: SendGrid
+  name: GOV.UK Notify
   description: 3rd party system to send e-mails
```

_double checking_
Is this description applicable for gov.uk notify too?

---

I guess it is external but not 3rd party but what's the cut of point when things become 3rd party (not authored by govuk?)?

---

Their page says

> Try GOV.UK Notify now if you work in central government, a local authority, or the NHS.

Based on this, what do you think about?

```suggestion
  description: Government-wide notification system (https://www.notifications.service.gov.uk/)
```